README.md

```diff
@@ -85,7 +85,11 @@ Rscript R/update-cases.R
 Rscript R/update-deaths.R
 ```
 
-4. Run each `R/update-country.R` script in turn.
+4. Run `Rscript R/run-region-updates.R`. 
+
+   Adding `--help` will show documentation and the command options. This can run all or some regions and subregions, control logging and exclude unstable regions. 
+   
+   Note: Currently regions are all country level but the region/subregion pattern could be reused on any level of parent/child geographic dataset.
```

Comment:
This is a good point and could be an extension in the future. 

I haven't tested running on smaller geographic UK scales but this could be an interesting set up estimates to support.